### ISSUE TITLE
[Question] How to init widget to full screen?

### ISSUE THREAD

**Issue by GeorgeMina** [thingsboard/thingsboard#2384]
Is it possible, via css/html or otherwise, to display a widget in full screen mode, without the need to press the 'full-screen' icon in the top right corner?

**Comment by Frozka** [comment]
You could resize your widget in your dashboard to cover whole dashboard.
Then under Customer -> Manage customer users -> your user ->

![image](https://user-images.githubusercontent.com/23498646/75032398-2c748600-54b1-11ea-90f0-8cb2b102180f.png)

**Comment by GeorgeMina** [comment]
> You could resize your widget in your dashboard to cover whole dashboard.
> Then under Customer -> Manage customer users -> your user ->
> 
> ![image](https://user-images.githubusercontent.com/23498646/75032398-2c748600-54b1-11ea-90f0-8cb2b102180f.png)

Hi Frozka. I have tried resizing the widget to the full dashboard, but when I cast it onto the WebView of my android application, it's still the same size. 

When I click on Customers, I don't see the "Manage customer users option". I haven't found anything that allows me to edit my profile either. What am I doing wrong?

![image](https://user-images.githubusercontent.com/49873468/75111836-d5081e80-5646-11ea-9dcb-12e883803c95.png)

**Comment by Frozka** [comment]
Im not exactly sure where this setting I gave on PE is, I only worked with CE.

But I assume you should go under:
Customer Groups (or Customer Hierarchy) -> All -> Then create A Test Customer if you don't already have.

When you have a Customer, create a user for that customer. And then click on that user and you should see the "Always full screen" button and "Default dashboard" selection.

Then when you login with user you just created, the dashboard should be full screen.

**Comment by GeorgeMina** [comment]
Ok, yes you were right about creating a new user. As a matter of fact my account is registered in "User Groups" as well. I've enabled the "always full screen" option as you suggested. 

The widgets on my phone still showed the toolbar and full screen buttons, and when I logged onto my tenant account, there was no side bar on the left and I was stuck in selecting dashboards only (but I've reverted to non-full screen mode). 

I guess the only thing I could do is to cover these buttons on my phone with a view on top of them.
